source/_integrations/philips_js.markdown

```diff
@@ -8,11 +8,14 @@ ha_release: 0.34
 ha_codeowners:
   - '@elupus'
 ha_domain: philips_js
+ha_config_flow: true
 ---
 
 The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API. Instructions on how to activate the API and if your model is supported can be found [here](http://jointspace.sourceforge.net/download.html). Note that not all listed, jointSPACE-enabled devices won't have JSON-interface running on port 1925. This is true at least for some models before year 2011.
 
-To add your TV to your installation, add the following to your `configuration.yaml` file:
+The preferred way to enable the integration is via **Configuration** -> **Integrations**.
+
+Alternatively a deprecated way to add your TV to your installationis via `configuration.yaml` file:
```

Comment:
When an integration moves to a configuration flow, YAML configuration should be deprecated and thus removed from the documentation.